packages/expo-file-system/src/FileSystem.ts

```diff
@@ -313,3 +313,16 @@ export class DownloadResumable {
     this._subscription = null;
   }
 }
+
+/**
+ * Android only
+ */
+export async function askForDirectoryPermissionsAsync(
```

Comment:
```suggestion
export async function requestDirectoryPermissionsAsync(
```

we have used `request` elsewhere for permissions, i think it likely makes most sense to do the same here, unless there was an intentional reason to use `askFor` instead